src/App.jsx

```diff
@@ -1,15 +1,16 @@
 import Home from "./components/Home";
 import { Route, Routes } from "react-router-dom";
-import Sidebar from "./components/Sidebar/Sidebar";
+import SidebarNav from "./components/Sidebar/Sidebar";
+import SingleArticle from "./components/SingleArticle/SingleArticle";
 
 function App() {
-
   return (
     <>
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path="/:article_id" element={<SingleArticle />} />
```

Comment:
Good work implementing the route.

URLs usually have a path which clearly explains what they lead to. At the moment my address bar just says `/34` for example for article id 34, consider making it clearer by making the route `/articles/:article_id` instead, this helps the user understand where they are and what the current url path is